src/App.vue

```diff
@@ -164,6 +164,8 @@ a {
   font-size: 30px;
   color: #3a416f;
   outline: none;
+  -webkit-appearance: none;
+  appearance: none;
```

Comment:
Is the non-prefixed one necessary? As far as I know, no browser supports non-prefixed `appearance` yet (at least Safari doesn't). I may be wrong.

Reply:
Not necessary but there for future support

Reply:
Cool 👍 Thanks!